apl/tabular-operators/sample-operator.mdx

```diff
@@ -1,29 +1,146 @@
+description: 'This page explains how to use the sample operator function in APL.'
 ---
 
-Psuedo-randomly selects rows at a rate specified by a parameter. The sampling algorithm is not statistically rigorous but provides a way to explore and understand a dataset. For statistically rigorous analysis [summarize](/apl/tabular-operators/summarize-operator) should be used.
+The `sample` operator in APL psuedo-randomly selects rows from the input dataset at a rate specified by a parameter. This operator is useful when you want to analyze a subset of data, reduce the dataset size for testing, or quickly explore patterns without processing the entire dataset. The sampling algorithm is not statistically rigorous but provides a way to explore and understand a dataset. For statistically rigorous analysis, use `summarize` instead.
 
-## Syntax
+You can find the `sample` operator useful when working with large datasets, where processing the entire dataset is resource-intensive or unnecessary. It’s ideal for scenarios like log analysis, performance monitoring, or sampling for data quality checks.
+
+## For users of other query languages
+
+If you come from other query languages, this section explains how to adjust your existing queries to achieve the same results in APL.
+
+<AccordionGroup>
+<Accordion title="Splunk SPL users">
+
+In Splunk SPL, the `sample` command works similarly, returning a subset of data rows randomly. However, the APL `sample` operator requires a simpler syntax without additional arguments for biasing the randomness.
+
+<CodeGroup>
+```sql Splunk example
+| sample 10
+```
+
+```kusto APL equivalent
+['sample-http-logs'] 
+| sample 0.1
+```
+</CodeGroup>
+
+</Accordion>
+<Accordion title="ANSI SQL users">
+
+In ANSI SQL, there is no direct equivalent to the `sample` operator, but you can achieve similar results using the `TABLESAMPLE` clause. In APL, `sample` operates independently and is more flexible, as it’s not tied to a table scan.
+
+<CodeGroup>
+```sql SQL example
+SELECT * FROM table TABLESAMPLE (10 ROWS);
+```
+
+```kusto APL equivalent
+['sample-http-logs'] 
+| sample 0.1
+```
+</CodeGroup>
+
+</Accordion>
+</AccordionGroup>
+
+## Usage
+
+### Syntax
+
+```kusto
+| sample ProportionOfRows
+```
+
+### Parameters
+
+- `ProportionOfRows`: A float greater than 0 and less than 1 which specifies the proportion of rows to return from the dataset. The rows are selected randomly.
+
+### Returns
+
+The operator returns a table containing the specified number of rows, selected randomly from the input dataset.
+
+## Use case examples
+
+<Tabs>
+<Tab title="Log analysis">
+
+In this use case, you sample a small number of rows from your HTTP logs to quickly analyze trends without working through the entire dataset.
+
+**Query**
+
+```kusto
+['sample-http-logs']
+| sample 0.05
+```
+
+[Run in Playground](https://play.axiom.co/axiom-play-qf1k/explorer?initForm=%7B%22apl%22%3A%22%5B%27sample-http-logs%27%5D%20%7C%20sample%200.05%22%7D)
```

Comment:
![image](https://github.com/user-attachments/assets/282056f8-8840-4e06-b943-152d6225d4c8)

Reply:
This is a problem with the linter. sample takes a float. I can't tweak the example to avoid this message.